docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -47,7 +47,42 @@ uavcan.protocol.NodeStatus (Data Type ID = 341)
 The DroneCAN protocol requires that all nodes on a DroneCAN network periodically publish their status using the Node Status message. Vertiq modules support this behavior to conform 
 to the standard. A uavcan.protocol.NodeStatus message is published at 1 Hz during operation, reporting its health, current mode, and uptime.
 
-In normal operation a Vertiq module should always report its Health as OK, and its mode as Operational. No sub-modes or vendor specific status codes are currently supported.
+Vertiq's Node Health Indications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Vertiq modules support all four standard DroneCAN `health indications <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#:~:text=fatal%20malfunction.%0Auint2-,health,-%23%0A%23%20Current%20mode.%0A%23%0A%23%20Mode>`_. Modules enter each state given the following:
+
+Critical
```

Comment:
Possibly its obvious, but it may be good to specify that if any of these conditions are met its in this health state. It's potentially ambiguous right now if its all the conditions have to be true, or if just one of them. I think from context its probably obvious but it might be good to start off by saying something like "The module will enter Critical mode if any of the following conditions are true" or whatever. Something in each of these sections that makes it clear that you just need one to trigger it.

And that goes for all of these sections